statement.Rmd

```diff
@@ -2,28 +2,50 @@
+The purpose of the CAA Special Interest Group **Scientific Scripting Languages in Archaeology** is to provide an exchange platform for scientists in and around archaeology who use scripting languages to conduct major or minor parts of their research. Scientific programming in non-scripting languages is not excluded, but this SIG focussed on software projects and data analysis work-flows that use code interactively. 
 
-The Special Interest Group is an open format that is intended to support discussion, the circulation of new ideas, and the integration of different approaches in connection with the objectives. Everyone may participate on equal terms in accordance with the Code of Conduct, as is usual in open source software development.
+The Special Interest Group is an open format that is intended to support discussion, the circulation of new ideas, and the integration of different approaches in connection with the objectives. Everyone may participate on equal terms in accordance with the [Code of Conduct](...), as is usual in open source software development. To be as inclusive as possible and also for the sake of reproducibility we strongly prefer open source over proprietary software. 
+
+We especially invite students and young career researches to become part of a wider network of developers and users to improve their skills, and promote the use of scripting languages in research, publication and teaching in archaeology. Open and reproducible research is becoming increasingly important in all scientific disciplines. We have to avoid a "lost" generation of students in archaeology that can't keep pace with this development due to a lack of hard knowledge about computational tools and data exchange standards. The SIG aims to establish the career path of the research software engineer in archaeology.
 
 ### Scripting languages
 
 A scripting language is a programming language which allows live coding interaction with an software interpreter to perform tasks with data. It does not require the compilation of a software module (or only requires this incidentally to the main coding activity) and it is therefore linked to a rapid development style especially suitable for scientific data exploration, analysis, and visualisation. The following list includes popular scripting languages in archaeology and many other sciences:
 
 - [R](https://www.r-project.org/) 
 - [Python](https://www.python.org/)
+- [Bash](http://www.gnu.org/software/bash/)
```

Comment:
bash? Really?

Reply:
[Yess!](https://stackoverflow.com/questions/28693737/is-bash-a-programming-language) I would say it's even a pretty common language in data analysis pipelines. None of our projects works without some shell scripting.